lib/exporter.rb

```diff
@@ -276,7 +276,7 @@ def self.generate_composite_tiff(_coords, _origin, warpables, id, ordered)
       wid = "w" + warpable['id'].to_s
       geotiffs += ' ' + directory + wid + '.tif'
     end
-    gdalwarp = "gdalwarp -s_srs EPSG:3857 -te #{minlon} #{minlat} #{maxlon} #{maxlat} #{geotiffs} #{directory}#{id}.tif"
+    gdalwarp = "gdalwarp -s_srs EPSG:3857 -t_srs EPSG:4326 -te #{minlon} #{minlat} #{maxlon} #{maxlat} #{geotiffs} #{directory}#{id}.tif"
```

Comment:
OK, so this line is still wrong if 4326 isn't added. Checking in QGIS if 4326 fixes.

Reply:
OK, so changing it to 4326 busts our disk space. 900913 runs but puts it at null island.

Reply:
setting 3857 also put it at null island.

Reply:
OK - this works - `-te_srs EPSG:4326`

```suggestion
    gdalwarp = "gdalwarp -te_srs EPSG:4326 -t_srs EPSG:4326 -te #{minlon} #{minlat} #{maxlon} #{maxlat} #{geotiffs} #{directory}#{id}.tif"
```

Reply:
Great this generated a single z17 tile!

Reply:
It does eventually load in testing at https://mapknitter.org/maps/pvdtest with

```js
      L.tileLayer('https://4567-apricot-capybara-rlz9n7bb.ws-us18.gitpod.io/tms/1636991957b/{z}/{x}/{-y}.png', {}).addTo(map);
```

https://4567-apricot-capybara-rlz9n7bb.ws-us18.gitpod.io/tms/1636991957b/17/39538/82327.png

But, i get a 401 error. So a few things happening here.

Reply:
Despite 401 i think it's in the correct location, after running 

```
gdal2tiles.py -k --s_srs EPSG:900913 public/warps/1636991957/1636991957d.tif public/tms/1636991957b/
```